### PR TITLE
fix : removed event start date limit 

### LINKED ISSUE #6021: Wizard Step 1: Event Start Date Limit not needed
In the wizard step 1 there is a start date limit, that the start of the event has to be before the current date. But, actually there are use cases where the event should start before the current date, e.g. when the user wants to change the end date of an event after it has started. Then the user will receive the error message "event start date should be before current date". 

Considering this use case, please take out the requirement that the start date of an event has to be before the current date.

Screenshot from 2020-12-14 10-47-07

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #6021 

#### Short description of what this resolves:
Through this pr the limit in start date of an event got removed

#### Changes proposed in this pull request:

- removed the limit to start date of an event

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -464,9 +464,9 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     updateSalesEndDate(eventStartDate) {
       eventStartDate = moment(new Date(eventStartDate));
       this.data.event.tickets.forEach(ticket => {
-        if (moment(eventStartDate).isBefore(ticket.get('salesEndsAt'))) {
+        // if (moment(eventStartDate).isBefore(ticket.get('salesEndsAt'))) {
```
Comment (maze-runnar) [inline app/components/forms/wizard/basic-details-step.js:467]: don't comment code, delete it.

Comment (nlok5923) [reply app/components/forms/wizard/basic-details-step.js]: said changes done.

Comment (maze-runnar) [reply app/components/forms/wizard/basic-details-step.js]: need to be fixed on server too

Comment (nlok5923) [reply app/components/forms/wizard/basic-details-step.js]: I need to get familiarized with the server code base then

Comment (maze-runnar) [reply app/components/forms/wizard/basic-details-step.js]: you need to read issue again you have made totally different change what issue is saying

Comment (maze-runnar) [reply app/components/forms/wizard/basic-details-step.js]: And also i guess no frontend change is needed.

Comment (nlok5923) [reply app/components/forms/wizard/basic-details-step.js]: yes @maze-runnar i got it.

**Pull request closed without merging**